I PAWEL BROILO I MODULE ONBOARDING I WIREFRAME TO WEB CODE

PawelBroilo opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.
Wireframe Project consists of three articles in semantic HTML. Answers all requested questions. A typical Wireframe of a Website.
## Questions

Ask any questions you have for your reviewer.

cjyuan reviewed (commented):
Can you implement your change in `index.html`?

A well-prepared PR makes it easier for reviewers to approve it with minimal back-and-forth. The "PR Essentials" file on the [Canvas](https://codeyourfutur-yov6609.slack.com/archives/C07QX99JK7B) in the #cyf-code-review channel has some helpful tips to make your PRs more robust and ready for review.
Can you take a few minutes to check against the essentials, including how to prepare PR description? Doing so can help speed up the review process.

And as a practice to get better at using AI tools, try running your code by ChatGPT. You can ask ChatGPT
```
review HTML code:
...        <--- paste your HTML code here
```

Then take a look at the suggestions it gives you, and pick out the ones that actually make sense for your project. It's a great way to learn and improve your work.

cjyuan reviewed (commented):
1. You were supposed to modify `index.html` and `style.css` in the folder "Wireframe" instead of introducing new files. 

2. The webpage appearance and layout should match the wireframe. Here is an example of a webpage with a layout that matches the wireframe.
![image](https://github.com/user-attachments/assets/15f3fcb5-5a5a-4826-8e94-f8f0b716cb61)

3. In your PR description, the self-checklist boxes are not properly checked. Can you edit them to make them appear exactly like this:
![image](https://github.com/user-attachments/assets/0a4f47fb-62f3-49bd-80cf-bb2727c86c7e)

Please note that in CYF courses, the recommended way to inform the reviewer of your changes is to do both of the following:
  - Reply to their feedback.
    - In the responses, clarify how each piece of feedback was addressed to demonstrate that you've carefully reviewed the suggestions.
    - Your response may trigger a notification (depending on the reviewer's settings), helping ensure they’re aware of the updates you’ve made. 
  - Replace the "Reviewed" label by a "Needs review" label (you missed this step)

PawelBroilo commented:
Hi CJ,
I implemented correct Code for Wireframe in Review Changes Tab at GitHub.
I updated File Index.html and styles.css
Wireframe looks correct now.

cjyuan commented on Wireframe/Index.html:
```diff
@@ -0,0 +1,50 @@
+        <p>Learn about README files, wireframes, and Git branches.</p>
+    </header>
+    <main>
+        <section class="cards">
+            <div class="card card-large">
+                <div class="image-placeholder"></div>
+                <h2>What is the purpose of a README file?</h2>
+                <p>
+                    A README file introduces and explains a project. It provides important details such as how to install, run, and contribute to the project. It helps other developers understand the project's structure and purpose.
+                </p>
+                <a href="#" class="btn">Read more</a>
+            </div>
```
`<div>` has no semantic. Can you use `<article>` instead? (The requirements ask for three articles in the webpage).

cjyuan commented on Wireframe/Index.html:
```diff
@@ -0,0 +1,50 @@
+    <main>
+        <section class="cards">
+            <div class="card card-large">
+                <div class="image-placeholder"></div>
```
Could you add a placeholder `<img>` tag in your articles, so users can easily replace the image URL when using your template? You can use the `placeholder.svg` image in all three of your articles.

cjyuan commented:
You forgot to add the "Needs review" label.

cjyuan commented:
Both the code and the PR description were not modified based on my previous feedback.
No new commits was made on this branch since my previous review. 

If you have made any changes to your code, you need to push them to GitHub.

PawelBroilo commented:
I am completing my modification of Pull Request #553 and Pull Request #650
now. I will send all at 4pm at  latest like I mentioned on Slack and in an
email.

On Thu, 5 Jun 2025, 13:17 CJ Yuan, ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2943994844>
>
> Both the code and the PR description were not modified based on my
> previous feedback.
> No new commits was made on this branch since my previous review.
>
> If you have made any changes to your code, you need to push them to GitHub.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2943994844>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVFFPRWTYTBDYWN4OUA7XQD3CAYPBAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBTHE4TIOBUGQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

PawelBroilo commented:
Hi CJ,
I completed Wireframe Project. I implemented my Code in File Changed Tab at GitHub.
Files : Index.html and styles.css in PullRequest#553
I code with your Instructions on how to correct Wireframe to Code Project.
I implemented all corrections and Wireframe Project Code is in Files Changed Tab in GitHub.

PawelBroilo commented:
I implemented a correct Code to Wireframe Index.html File Its in Files
Changes at GitHub.

On Thu, 5 Jun 2025, 18:13 CJ Yuan, ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945333274>
>
> Page still does not appear correctly:
> https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945333274>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVFFPRUVI4IFQGKPJ7OMJSL3CB3E7AVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBVGMZTGMRXGQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

PawelBroilo commented:
It might an Image Photo.jpg path which is to internal Folder not Desktop. I
will see it now.

On Thu, 5 Jun 2025, 19:07 Pawel Broilo, ***@***.***> wrote:

> Hi CJ,
> I test this File on my PC. I then copy and paste my Code into GitHub. I
> don't see a reason for it not to show a correct content. I will resend it
> now.
>
> On Thu, 5 Jun 2025, 19:04 CJ Yuan, ***@***.***> wrote:
>
>> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945528330>
>>
>> Page still does not appear correctly:
>> https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/
>> You can see it yourself
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945528330>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AVFFPRW4C3Q7LSRL2YJWELT3CCA5ZAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBVGUZDQMZTGA>
>> .
>> You are receiving this because you modified the open/close state.Message
>> ID: ***@***.***>
>>
>

cjyuan commented:
Check the webpage yourself: https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/

You need to make sure this page is rendered correctly after your changes.

PawelBroilo commented:
I think its because my Photo.jpg Desktop image is not visible at Wireframe
Project at GitHub. It seems GitHub version of my Wireframe Project doesn't
display my Wireframe Image from my Desktop. I don't know how to fix it at
GitHub.Wireframe opens normal for me with Photo.jpg in its code on my PC.
Its not possible to paste an image and store it in GitHub. CJ I don't know
how to solve an Image issue in Wireframe Project at GitHub.

On Thu, 5 Jun 2025, 19:36 CJ Yuan, ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945631073>
>
> Check the webpage yourself:
> https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/
>
> You need to make sure this page is rendered correctly after your changes.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945631073>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVFFPRWWOAGPINOMCWTVFPL3CCEZJAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBVGYZTCMBXGM>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

PawelBroilo commented:
Hi CJ,
I updated PR#650 and PR#553.
New File FormControls.html passes Lighthouse Test. Its in Files Changed at
GitHub.
Photo of FormControls.html is included in Comments in Pull Request.
New File Wireframe.html is edited to be viewed in GitHub.
Photo of Wireframe.html is included in Comments at GitHub.
Its in Files Changed in Pull Reguest.
I edited both FormControls.html to pass Lighthouse test.
Wireframe.html so you can view it in GitHub.
Thank you

On Thu, 5 Jun 2025, 20:17 CJ Yuan, ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945798486>
>
> I have already suggested a solution several times. I am not going to
> repeat. You can search for it in the comments or ask ChatGPT to suggest a
> solution.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2945798486>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVFFPRSTNFP5NPI5EYG4YRD3CCJVHAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBVG44TQNBYGY>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

cjyuan commented on Wireframe/Wireframe.html:
```diff
@@ -0,0 +1,50 @@
+            <article class="article2">
+                <section>
+                    <img src="./Photo.jpg" alt="Readme Example" class="full-width-image">
+                    <p>What is the purpose of a README file?</p>
```
This should be a header and not a paragraph.

cjyuan reviewed (commented):
1. This is what your webpage looks like when deployed on Netlify: 
https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/wireframe
It does not work properly because the server cannot find the image file `Photo.jpg` in your Github repository.

2. You were supposed to prepare your code in `index.html` and `style.css`. If you are having difficulty renaming the files, you can copy your code to those files.

3. Can you improve the Lighthouse Accessibility score of your page to 100?

4. One of the acceptance criteria, "The page footer is fixed to the bottom of the **viewport**", has not yet been satisfied. Can you make the necessary change? (Suggestion: Ask ChatGPT what that requirement means or look up the term "viewport").

5. Can you format the checked boxes using the proper Markdown syntax in your PR description so that they look something like this?
![image](https://github.com/user-attachments/assets/6f85e6c5-4dd7-49aa-a9fc-e17a9dc2189d)
(With proper Markdown syntax, we can use mouse to check/uncheck the items)

PawelBroilo reviewed (commented):
[file:///C:/Users/pawel/OneDrive/Desktop/CYF/Wireframe/index.html](url)
![wireframe](https://github.com/user-attachments/assets/2e5ef8a2-539a-4bf8-9f8b-f0950345bd57)
[styles.css](https://github.com/user-attachments/files/20634189/styles.css)

cjyuan commented:
`file:///C:/Users/pawel/OneDrive/Desktop/CYF/Wireframe/index.html`
Can you find out from ChatGPT why other people cannot open the above URL?

You need to add those files to your repository on Github or else I cannot access the files.

cjyuan commented:
If you clicked the "Files changed" tab in this PR (https://github.com/CodeYourFuture/Module-Onboarding/pull/553/files), you can see this
![image](https://github.com/user-attachments/assets/85379c10-afe1-4879-bbbe-96ff0b93bdfd)
- It means, the CSS file `style.css` in the "Wireframe" folder has been deleted
- The `wireframe.jpg` is added to the top-level folder.
- A new `style.css` has been added to the top-level folder.

As a result, this is how your webpage looks like when it is deployed:
https://deploy-preview-553--cyf-onboarding-module.netlify.app/wireframe/

PawelBroilo commented:
I realised exactly this order of current Files in my Wireframe Project. I think how to modify these Files to display my Wireframe Project correct because the code in those Files is correct. I think maybe its better to prepare a New Pull Request with Wireframe Project. I try to implement a new Pull Request  now.

cjyuan commented:
A new PR sounds good.

PawelBroilo commented:
New PR#716 displays my Wireframe Project on Netlify Deploy. I think its
excellent. It looks exactly like on my PC. I hope Code Your Future can
apporve it.

On Sat, 7 Jun 2025, 20:36 CJ Yuan, ***@***.***> wrote:

> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>
>
> A new PR sounds good.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVFFPRUMDQIIDAIE5ULTYRT3CM5MJAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNJSHEZDIOJZGU>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

PawelBroilo commented:
Hi CJ,
I.deleted Repository Branch for Pull Request #650 Form Controls and Pull
Request#650 Form Controls is now Closed and it is "Completed"
CJ can you reopen my Pull Request #650 via GitHub?

On Sat, 7 Jun 2025, 21:24 Pawel Broilo, ***@***.***> wrote:

> New PR#716 displays my Wireframe Project on Netlify Deploy. I think its
> excellent. It looks exactly like on my PC. I hope Code Your Future can
> apporve it.
>
> On Sat, 7 Jun 2025, 20:36 CJ Yuan, ***@***.***> wrote:
>
>> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>
>>
>> A new PR sounds good.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AVFFPRUMDQIIDAIE5ULTYRT3CM5MJAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNJSHEZDIOJZGU>
>> .
>> You are receiving this because you modified the open/close state.Message
>> ID: ***@***.***>
>>
>

PawelBroilo commented:
I deleted Repository Branch for Pull Request #650 Form Controls and Pull
Request#650 Form Controls is now Closed and it is "Completed"
CJ can you reopen my Pull Request #650 via GitHub?

On Sat, 7 Jun 2025, 21:33 Pawel Broilo, ***@***.***> wrote:

> Hi CJ,
> I.deleted Repository Branch for Pull Request #650 Form Controls and Pull
> Request#650 Form Controls is now Closed and it is "Completed"
> CJ can you reopen my Pull Request #650 via GitHub?
>
> On Sat, 7 Jun 2025, 21:24 Pawel Broilo, ***@***.***>
> wrote:
>
>> New PR#716 displays my Wireframe Project on Netlify Deploy. I think its
>> excellent. It looks exactly like on my PC. I hope Code Your Future can
>> apporve it.
>>
>> On Sat, 7 Jun 2025, 20:36 CJ Yuan, ***@***.***> wrote:
>>
>>> *cjyuan* left a comment (CodeYourFuture/Module-Onboarding#553)
>>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>
>>>
>>> A new PR sounds good.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/CodeYourFuture/Module-Onboarding/pull/553#issuecomment-2952924995>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AVFFPRUMDQIIDAIE5ULTYRT3CM5MJAVCNFSM6AAAAAB5LX3CAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNJSHEZDIOJZGU>
>>> .
>>> You are receiving this because you modified the open/close state.Message
>>> ID: ***@***.***>
>>>
>>
